packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -212,7 +212,8 @@ public Future<DocumentAsset> extractPDF(
 							final ObjectMapper objectMapper = new ObjectMapper();
 
 							final JsonNode rootNode = objectMapper.readTree(bytes);
-							if (rootNode instanceof final ArrayNode arrayNode) {
+							if (rootNode instanceof ArrayNode) {
```

Comment:
Just an FYI - java 17 has some neat improvements here that let you get rid of the casting statement and just do something like:



```
if (rootNode instanceof ArrayNode arrayNode){
   for (final JsonNode record : arrayNode)
   
   ...
}
```